### ISSUE TITLE
VIEWS AND FILTERS

### ISSUE THREAD

**Issue by JoachimGeens** [Maradonna90/NewGAN-Manager#215]
Can't find the view folder in mijn FM 24 game. I did follow the instructions. 

![Schermafbeelding 2024-01-10 155133](https://github.com/Maradonna90/NewGAN-Manager/assets/156212825/dc65c0db-4de1-43a1-bace-36a97f2bca83)

**Comment by XanUtd** [comment]
The view folder is in the RARA file you downloaded. Need to copy and paste it to the FM folder

**Comment by JoachimGeens** [comment]
Yes I did this.



Verzonden vanaf mijn Galaxy



-------- Oorspronkelijk bericht --------
Van: XanUtd ***@***.***>
Datum: 10/01/2024 16:37 (GMT+01:00)
Aan: Maradonna90/NewGAN-Manager ***@***.***>
Cc: Joachim Geens ***@***.***>, Author ***@***.***>
Onderwerp: Re: [Maradonna90/NewGAN-Manager] VIEWS AND FILTERS (Issue #215)


The view folder is in the RARA file you downloaded. Need to copy and paste it to the FM folder

**Comment by Ayal1** [comment]
Are you sure? Send a screenshot of your FM24 folder and the views folder

**Comment by rogeriofza** [comment]
Is there a filter for Staff new gens...would be nice to do staff faces also

**Comment by Ayal1** [comment]
> Is there a filter for Staff new gens...would be nice to do staff faces also

No, or at least there's no obvious way of adding one if it does exist. SI would need to implement something for that to work.
